### DISCUSSION TITLE
Using a climatological currents file?

### DISCUSSION THREAD

**Discussion by AndresSepulveda** [OpenDrift/opendrift#506] [Ideas]
Hi,

We want to do larvae release studies using climatologial model results (every year  the forcing to produce these files is the same), a file with just one year.

If I release particles in December, is the a way for OpenDrift to use the currents from January, insteat of adding another Janaury at the end of the ocean model files?

In one case the lifetime of the particles is 3 months, so instead of using a 12+3 months file, I could use only a 12 months file.

Regards,

Andrés

**Comment by knutfrode** [comment]
Hi,

In this climatological file, is there a temporal resolution (e.g. hourly, daily or monthly), or is is just an average current for the whole year (single time)?
If it is a single time (yearly average), then you could make it valid "forever" by typing ```r.always_valid = True```

If it has several time steps, then I'm afraid there is no other solution than to manipulate your forcing file by repeating each year successively. This could probably be done fairly easily by Nco or CDO, though you might get a fairly large output file.
